xiloader/network.cpp

```diff
@@ -333,6 +338,57 @@ namespace xiloader
             closesocket(sock->s);
             sock->s = INVALID_SOCKET;
             return false;
+
+        case 0x0005: // Request for updated password to change to.
+        {
+            xiloader::console::output(xiloader::color::success, "Log in verified for user %s.", g_Username.c_str());
+            std::string confirmed_password = "";
+            do
+            {
+                std::cout << "Enter new password (6-15 characters): ";
+                g_Password.clear();
+                std::cin >> g_Password;
+                std::cout << "Repeat Password           : ";
+                std::cin >> confirmed_password;
+                std::cout << std::endl;
+
+                if (g_Password != confirmed_password)
+                {
+                    xiloader::console::output(xiloader::color::error, "Passwords did not match! Please try again.");
+                }
+            } while (g_Password != confirmed_password);
```

Comment:
This would work, I wonder if it would be worthwhile to have a max tries counter.

Reply:
That's a good idea. I'll leave this comment open and take a stab at that when i get some free time.